src/forward_sensitivity.jl

```diff
@@ -158,7 +158,7 @@ function ODEForwardSensitivityProblem(f::F, args...; kwargs...) where {F}
 end
 
 function ODEForwardSensitivityProblem(prob::ODEProblem, alg; kwargs...)
-    ODEForwardSensitivityProblem(prob.f, prob.u0, prob.tspan, prob.p, alg; kwargs...)
+    ODEForwardSensitivityProblem(symbolic_container(prob), state_values(prob), prob.tspan, parameter_values(prob), alg; kwargs...)
```

Comment:
`symbolic_container` technically does returns `prob.f`, however semantically it isn't guaranteed to do so and in such cases where you explicitly wand the function, I'd still use `prob.f`.

Reply:
This has been addressed